.gitignore

```diff
@@ -92,6 +92,6 @@ nuiEditorAutosave.json
 # Ignore stuff we extract for use with Terasology
 /natives/
 config/metrics/
-
+configs/engine/rendering.cfg
```

Comment:
Must have sneaked in through a commit 😄 We do need this, but I am not sure if this PR is the right place for the change.

Reply:
@eviltak Yeah I noticed that. It came with my earlier particle javadocs commit that this was based on, a mentor requested that I add that in from an earlier task I think.

Reply:
This change has been incorporated in #3227 by eviltak@f393d2f, so it might be better to remove this change here to prevent merge conflicts.